MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -5,6 +5,8 @@
 import UIKit
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching {
+	@IBOutlet public var errorView: ErrorView!
```

Comment:
While having `errorView` getter public it is still important to have the setter `private` to prevent clients from changing the instance. Please make the setter private while leaving the getter public

```suggestion
	@IBOutlet public private(set) var errorView: ErrorView!
```

Reply:
Done

Reply:
But construction public private(set) - drive me crazy ;)